types/google-spreadsheet/index.d.ts

```diff
@@ -72,15 +72,15 @@ export interface WorksheetGridRange {
     endColumnIndex: number;
 }
 
-export interface WorksheetGridProperties {
+export type WorksheetGridProperties = Partial<{
```

Comment:
The problem with using `Partial<{ ... }>` instead of an interface is that error elaborations can become very messy:

![image](https://user-images.githubusercontent.com/3902892/95641399-e9265200-0a56-11eb-8888-8db5ddd5a4aa.png)

vs.

![image](https://user-images.githubusercontent.com/3902892/95641376-c3994880-0a56-11eb-8872-0925aefbef88.png)

I will approve and leave it up to you if it should merge as-is or if you want to change the definition.

Reply:
good catch there man. i see the value of the interface approach in this context. 

i'm not a TS dev so @EndBug  worked with me on making these. i'll defer the decision to him.

Reply:
My mistake, I'll switch back to an interface 👍